ts/Extensions/DataGrouping.ts

```diff
@@ -736,6 +759,9 @@ const baseProcessData = seriesProto.processData,
         },
         ohlc: {
             groupPixelWidth: 5
+        },
+        hlc: {
+            groupPixelWidth: 10
```

Comment:
Why `10`? The `ohlc` has `groupPixelWidth: 5`?

Reply:
It looked better for me. I can change it to 5